### ISSUE TITLE
Not working in fw 12.0.0 with atmosphere 19.0

### ISSUE THREAD

**Issue by giorgioshine** [cathery/sys-con#246]
After installation and reboot, it detects the Xbox one s controller but it doesn't work.

**Comment by Arek75** [comment]
I was about to post this myself.  The reason is that because of the changes in 12.0.0 and the resultant required changes in AMS 0.19.0, anything using Atmosphere extensions (mitm), needs to be rebuilt against the new AMS.  I'd do it myself but am unsure what all I'd need to do it.

--Arek75

**Comment by cathery** [comment]
I pushed out a new release that seems to work on AMS 0.19.0 and FW 12.0.0. I didn't get to test it out completely though.

**Comment by giorgioshine** [comment]
I tried the new build and it crashes on boot.

```
Panic occurred while running Atmosphere

Title ID: 690000000000000D
Error: std::abort (0xFFE)

Report saved to atmosphere/fatal_errors/report_0b7d132a.bin

Press POWER to continue.
```

I attached the crash report.
[report_0b7d132a.bin.zip](https://github.com/cathery/sys-con/files/6295322/report_0b7d132a.bin.zip)

**Comment by cathery** [comment]
@giorgioshine That's a really weird crash. Is your Atmosphere up to date? Are you running any other sysmodules? Does it happen every time you boot?

**Comment by giorgioshine** [comment]
@cathery I'm running atmosphere 0.19.0 on fw 12.0 and no other sysmodules. After installing 0.6.3 it crashes every boot, it doesn't happen with 0.6.2 tho

**Comment by cathery** [comment]
Okay, I reproduced it too. I'll see if I can fix it

**Comment by cathery** [comment]
It turns out I was testing on fw 11 the entire time. I managed to fix your issue locally @giorgioshine, but there are more things not fully supported on fw 12 and it'll take some time before I can update my project.

**Comment by giorgioshine** [comment]
oh I see, take your time, and thanks for the amazing project btw

**Comment by FaserF** [comment]
Just to give some feedback: It also crashes with AMS 0.19.1 and FW11.0.1 with the latest dev sys-con version.

**Comment by cathery** [comment]
Released 0.6.3 - tested working on FW 12.0.0 and FW 11.0.1

**Comment by Sam-Man16** [comment]
> Released 0.6.3 - tested working on FW 12.0.0 and FW 11.0.1

On AMS 19.0 and 19.1 my device won't boot (no crashes) even with v0.6.3. I have no idea what the issue is.

**Comment by giorgioshine** [comment]
Ok I just tried the new update. It works fine in hand-held mode, but not in docked. I tried every port and it just doesn't detect it. I've got an Xbox one controller btw

**Comment by FaserF** [comment]
> > Released 0.6.3 - tested working on FW 12.0.0 and FW 11.0.1
> 
> On AMS 19.0 and 19.1 my device won't boot (no crashes) even with v0.6.3. I have no idea what the issue is.

For me it is working without issues on FW11.0.1 and 0.6.3
